main.tf

```diff
@@ -76,6 +82,8 @@ resource "google_project" "project" {
   auto_create_network = "${var.auto_create_network}"
 
   labels = "${var.labels}"
+
+  app_engine = "${local.app_engine_config["${local.app_engine_enabled ? "enabled" : "disabled"}"]}"
```

Comment:
@ryanckoch This is hard to parse and somewhat confusing. 

Wouldn't `app_engine = "${local.app_engine_enabled ? var.app_engine : list()}"` work just as well?

Reply:
It wont allow for a list in a conditional. That would definitely be much cleaner, but that is the reason for the hack app_engine_config map.

Reply:
How about `"${local.app_engine_enabled ? local.app_engine_config["enabled"] : local.app_engine_config["disabled"]}"`

Reply:
Same:
```
Error: module.project-factory.google_project.project: 1 error(s) occurred:

* module.project-factory.google_project.project: At column 3, line 1: conditional operator cannot be used with list values in:

${local.app_engine_enabled ? local.app_engine_config["enabled"] : local.app_engine_config["disabled"]}
```